.github/workflows/ci-build.yml

```
@@ -3,35 +3,43 @@ name: CI Build
 
 on:
   push:
-    branches: [ main, v2 ]
+    branches:
+      - main
+      - v2
   pull_request:
 
 jobs:
   build:
-    # Avoiding -latest due to https://github.com/actions/setup-python/issues/162
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

The latest version may not support 3.7 🤔 

```suggestion
    runs-on: ubuntu-22.04
```

---

@WilliamBergamin Nice catch and thanks for the fast fix! I missed this altogether... 🤖